ssc/ssc_equations.h

```diff
@@ -84,22 +84,21 @@ struct ssc_equation_entry{
 static ssc_equation_entry ssc_equation_table [] = {
         // Marine energy
         
-		{"me_array_cable_length", me_array_cable_length,
-            "Marine Energy", me_array_cable_length_doc,
+        {"me_array_cable_length", me_array_cable_length,
+            "MhkWave", me_array_cable_length_doc,
```

Comment:
I think this should be attached to the MhkCost module, since the outputs of these equations are inputs to MhkCost, not MhkWave or MhkTidal. Thoughts?